tsconfig.json

```diff
@@ -10,10 +10,12 @@
         "preserveConstEnums": true,
         "esModuleInterop": true,
         "sourceMap": true,
-        "outDir": "./dist"
+        "outDir": "./dist",
+        "skipLibCheck": true,
```

Comment:
it seems to build correctly without setting `skipLibCheck` to `true`. Why is that flag here?